docs/getting-started/getting-started.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: What you can use Patrol for?
```

Comment:
I'd prefer to have this name (and the `#` heading below) match the name of the page in `docs.json`.